torchvision/ops/poolers.py

```diff
@@ -9,40 +10,6 @@
 from .roi_align import roi_align
 
 
-# copying result_idx_in_level to a specific index in result[]
-# is not supported by ONNX tracing yet.
-# _onnx_merge_levels() is an implementation supported by ONNX
-# that merges the levels to the right indices
-@torch.jit.unused
-def _onnx_merge_levels(levels: Tensor, unmerged_results: List[Tensor]) -> Tensor:
-    first_result = unmerged_results[0]
-    dtype, device = first_result.dtype, first_result.device
-    res = torch.zeros(
-        (levels.size(0), first_result.size(1), first_result.size(2), first_result.size(3)), dtype=dtype, device=device
-    )
-    for level in range(len(unmerged_results)):
-        index = torch.where(levels == level)[0].view(-1, 1, 1, 1)
-        index = index.expand(
-            index.size(0),
-            unmerged_results[level].size(1),
-            unmerged_results[level].size(2),
-            unmerged_results[level].size(3),
-        )
-        res = res.scatter(0, index, unmerged_results[level])
-    return res
-
-
-# TODO: (eellison) T54974082 https://github.com/pytorch/pytorch/issues/26744/pytorch/issues/26744
-def initLevelMapper(
-    k_min: int,
-    k_max: int,
-    canonical_scale: int = 224,
-    canonical_level: int = 4,
-    eps: float = 1e-6,
-):
-    return LevelMapper(k_min, k_max, canonical_scale, canonical_level, eps)
```

Comment:
I understand you moved both methods at the bottom without modifications, correct? If yes then I agree that it's best to define them after the method.

Could you separate the refactoring of this file on a separate PR, so that we make the reviewing easier?